### ISSUE TITLE
error linking lnewt 0.52.19 with libnewt-dev 0.52.20

### ISSUE THREAD

**Issue by tuxd3v** [tuxd3v/lnewt#1]
newtListbox - Not defined

/usr/lib/lua/5.3/lnewt.so: undefined symbol: newtListbox

**Comment by tuxd3v** [comment]
seems that in libnewt 0.52.20,
The symbol only exists in the statically linked object:
```lua
~# grep -rni "newtListbox" /usr/lib --color
Binary file /usr/lib/aarch64-linux-gnu/libnewt.a matches
```
*Not* in Dynamically linked '/usr/lib/aarch64-linux-gnu/libnewt.so.0.52.20'

**Comment by tuxd3v** [comment]
This problem has being solved with 0.52.21 release which will be in master.
Case closed..
It was the way the project needed to be built..
